tensorboard/webapp/plugins/plugins_component.ts

```diff
@@ -135,13 +148,17 @@ export class PluginsComponent implements OnChanges {
   private readonly pluginInstances = new Map<string, HTMLElement>();
 
   ngOnChanges(change: SimpleChanges): void {
+    if (!this.isFeatureFlagsLoaded || !this.activeKnownPlugin) {
```

Comment:
I think it technically is possible for the app to go from truthy `this.activeKnownPlugin` to falsy `this.activeKnownPlugin` when you have experiments (navigate from an exp with data to one without data). I think it is more correct to hide the previous activeKnownPlugin in that case. WDYT?

Reply:
I don't think the ngOnChanges() implementation at HEAD does anything when this.activeKnownPlugin is falsy. Is this a case we hadn't previously considered or am I introducing a bug?  Would I have to reproduce in tensorboard.corp?

Reply:
True, I thought L160 would have handled that case, but it did not (i.e., your code does not cause regression). Could you leave a TODO or, if you think it is small enough change, just fix the bug?

Reply:
Added TODO.